AUTHORS

```diff
@@ -20,7 +20,6 @@ Major contributing authors include:
 - Tony Locke
 - Hajime Nakagami
 - Vraj Mohan
-- Mike Bayer
 - Robert Leftwich
 - Taavi Burns
 - jonathan vanasco
```

Comment:
Also, this is random, but I'm not sure if the first and last name should be capitalized here?

Reply:
sure that looks wrong, he uses capitals

Reply:
ill put this one in for now thanks!